### ISSUE TITLE
2.1.2 throwing ' Invalid value for <path> attribute d="

### ISSUE THREAD

**Issue by dizzy0ny** [PMSI-AlignAlytics/dimple#145]
In the below example, i am seeing a lot of errors like this: "Error: Invalid value for <path> attribute d="M312.5000000002631,30.192106079999995L312.5000000002631,335LM""

http://jsfiddle.net/z6px1kzk/

I think as a consequence of these errors, the circle on the second series is not hidden.
This does not happen with older versions of dimple such as dimple.v1.1.5.min.js
Any suggestions on how to work around this issue?

**Comment by dizzy0ny** [comment]
any thoughts on what is causing this?  don't mind if i have to fix by modifying source for now

**Comment by johnkiernander** [comment]
I can't look at this right now but I recently suggested a better approach for a vertical line in this stack overflow article: http://stackoverflow.com/questions/29352970/dimple-js-add-vertical-line/29358705#29358705

This might help?

**Comment by dizzy0ny** [comment]
Yes, but having difficulties trying to adapt it to a TimeAxis..where my x axis values are dates.  How would i translate a string date to the appropriate values that need to be set for x1 and x2?

**Comment by johnkiernander** [comment]
It just needs to be a JavaScript date. So you can use:

    new Date("2015-04-13")

Or for a more reliable approach use d3:

    d3.time.format("%Y-%M-%d").parse("2015-04-13")

**Comment by dizzy0ny** [comment]
I may be missing something. do i need to convert that date into a position on the x-axis?  seems as though it is drawing it off the grid/chart:  https://jsfiddle.net/z6px1kzk/

**Comment by johnkiernander** [comment]
Yes that's exactly what was missing, once draw has been called you can access the underlying d3 scale of an axis in `x._scale`.  So I just moved the line drawing after the chart drawing, added the x._scale and it works: http://jsfiddle.net/gk3jkwtt/1/
